Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include "config.h"
 #include "MediaPlayerPrivateGStreamer.h"
+#include <gst/gstinfo.h>
```

Comment:
This was inserted by LSP I guess? :) Not needed

Reply:
Argh! 🤦 Thanks! 🙏